lib/routemap.c

```diff
@@ -1281,14 +1292,39 @@ const char *route_map_get_match_arg(struct route_map_index *index,
 	return (NULL);
 }
 
+static int8_t get_route_map_delete_event(route_map_event_t type)
```

Comment:
change the return to a `route_map_event_t` and assert right before the return of -1, then fix the caller of this function

Reply:
Thanks @donaldsharp. I've made the changes.